keep-ui/shared/ui/YAMLCodeblock/ui/MonacoYAMLEditor.tsx

```diff
@@ -444,7 +446,7 @@ const MonacoYAMLEditor = ({
         workflow: Record<string, unknown>;
       };
       // update workflow
-      await updateWorkflow(workflowId!, parsedYaml);
+      await updateWorkflow(workflowId, parsedYaml, { mutateWorkflowDetail });
```

Comment:
it's better not to pass around `mutateWorkflowDetail`, but instead call mutateMultiple with appropriate key `workflows/${workflowId}` in useWorkflowActions()